internal/interceptor/server_test.go

```diff
@@ -565,3 +575,17 @@ func readTestdataFile(t *testing.T, filename string) []byte {
 	}
 	return b
 }
+
+// hmacHeader generates a X-Hub-Signature header given a secret token and the request body
+// See https://developer.github.com/webhooks/securing/#validating-payloads-from-github
+// Note that while this example and the validation comes from GitHub, it applies to
+// Bitbucket just the same.
+func hmacHeader(t *testing.T, secret string, body []byte) string {
+	t.Helper()
+	h := hmac.New(sha1.New, []byte(secret))
```

Comment:
If I'm interpreting https://confluence.atlassian.com/bitbucketserver/manage-webhooks-938025878.html correctly, Bitbucket is using SHA256 as a hashing algorithm to generate the HMAC.

Reply:
Thanks for checking, fixed.